vite/features.ts

```diff
@@ -102,8 +103,6 @@ const features = () => {
     features._census[census] = true
   }
 
-  console.log('features:', features)
```

Comment:
WHY? Get it back please. Change it to console.info if you feel better about it, but that console has to be there.